tests/__init__.py

```diff
@@ -4,40 +4,31 @@
 import unittest
 import numpy as np
 
-def for_all_versions(version_map: dict, runtype_map: dict):
-    def decorator(test_function):
-        @functools.wraps(test_function)
-        def inner(self):
-            for version_test_label, version in version_map.items():
-                for job_test_label, jobtype in runtype_map.items():
-                    with self.subTest(version_number=version_test_label, job_type=job_test_label):
-                        test_function(self, version, jobtype)
-        return inner
-    return decorator
+
+@functools.lru_cache(maxsize=1)
+def get_test_manifest():
+    rootDir = os.path.dirname(__file__)
+    with open(os.path.join(rootDir, "manifest.yaml")) as fp:
+        manifest = yaml.load(fp, Loader=yaml.FullLoader)
+    return manifest
+
 
 class TestBase(unittest.TestCase):
     """
     Base class for express tests.
     """
+    manifest = get_test_manifest()
```

Comment:
Maybe just define it as a class property/function to avoid doing `**TestBase.manifest` below, no?